tests/components/zha/common.py

```diff
@@ -40,8 +40,9 @@ def add_input_cluster(self, cluster_id, _patch_cluster=True):
         if _patch_cluster:
             patch_cluster(cluster)
         self.in_clusters[cluster_id] = cluster
-        if hasattr(cluster, "ep_attribute"):
-            setattr(self, cluster.ep_attribute, cluster)
+        ep_attribute = getattr(cluster, "ep_attribute")
```

Comment:
Using `getattr` with a static string is no different than using it directly.

```suggestion
        ep_attribute = cluster.ep_attribute
```

Reply:
Looking at this logic change, are you sure this is correct?

If the attribute now is missing it will throw an exception...
Previously it checked if the attribute existed, now it only checks if it is an empty/none value.

Reply:
yeah, you are right. The attribute could be missing in the previous versions, but now it is always there, which makes this entire check moot.
I'll update.

Reply:
actually this entire logic is outdated, we do rely on all clusters having the ep_attribute for a while now.